### DISCUSSION TITLE
Error using dwi2cond to check ernie dataset

### DISCUSSION THREAD

**Discussion by savvy73** [simnibs/simnibs#279] [9. Results and analysis]
I have been trying to run the ernie dataset using charm and get the anisotropy of the data using dwi2cond as given in the following page -https://simnibs.github.io/simnibs/build/html/documentation/command_line/dwi2cond.html. When running dwi2cond -check ernie, I am getting an error as follows - ERROR: No option given. Consider to rerun with option --prepro
and when I run using prepro I get the following using preprocessed DTI tensor (ignoring other preprocessing options) ...
 ```
==============================
==> dwi2cond started  Wed Jan 24 10:31:02 CET 2024 ...
==============================
 
==============================
==> --prepro: Preprocessing DWI data ...
==============================
Copying DTI tensor
ERROR: Input file '--prepro' doesn't exist.
 ```
Can someone provide any suggestions on how to correct this ?

**Comment by gruberanthony** [comment]
I would recommend deleting the initial files created from the first run of dwi2cond and then trying again using --prepreo added. It is weird that the dwi2cond command didn't run preprocessing on your initial run. 

Do you mind also adding your initial command to create the dti files after running dwi2cond. That could also help to understand what may be causing the error.
